src/server/object_services/md_store.js

```diff
@@ -1389,6 +1414,17 @@ class MDStore {
         });
     }
 
+    // Only for clean up in testing - Don't use unless you are sure!!!!
+    async delete_all_chunks_in_system() {
+        const delete_date = new Date();
+        return this._chunks.col().updateMany({}, {
+            $set: {
+                deleted: delete_date,
+                reclaimed: delete_date
```

Comment:
chunks don't have a `reclaimed` property in the schema... 
What did you mean to do here?